alerts/plugindefinition.yaml

```
@@ -38,6 +38,14 @@ spec:
       required: false
       type: list
 
+    - name: alerts.prometheusOperator.enabled
+      description: Deploy Prometheus Operator if kube-monitoring has not already installed it once.
+      required: false
+      type: bool
+    - name: alerts.crds.enabled
+      description: Install Prometheus Operator CRDs if kube-monitoring has not already installed them.
+      required: false
+      type: bool
```

>Yes. We enable it by default in the chart values, but disable it by default for the plugin as we intentionally want >kube-monitoring to manage the operator and the CRDs. These options only highlight the possibility for the >plugin to act independently

@richardtief, Thank you for the explanation. Do we need to set this default value to `false` explicitly like below then? 
Or, Am I missing something? 
```suggestion
      required: false
      default: false
      type: bool
```